Riot/Views/RoomBubbleList/RoomIncomingTextMsgBubbleCell.xib

```diff
@@ -1,11 +1,11 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="11762" systemVersion="15G1108" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES">
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The room bubble views are very tricky, we should not update this kind of view in this first PR. We will handle them later in another PR.

Please remove any changes applied inside the "RoomBubbleList" directory for this PR.

As you can see in the following screenshot with a large text size, you forgot to update some views (small username), and updated views layout is not correct (username and message are too close):

![screen shot 2017-10-30 at 19 22 10](https://user-images.githubusercontent.com/8969772/32188299-b4a6b546-bda7-11e7-8c4d-79d88be753f9.png)